rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -277,7 +278,7 @@ def __init__(
         """Declare instance variables with default values."""
 
         if component_config is not None and EPOCHS not in component_config:
-            logger.warning(
+            raise_warning(
```

Comment:
optional: it'd be a bit nice importing the module, but I guess this is the way we have done it everywhere and for things like that it still makes sense

Reply:
oh of course, how could i forget 😄

Reply:
I guess i'll leave it for now